server/config.go

```diff
@@ -286,35 +286,27 @@ type ScheduleConfig struct {
 	// a store will be considered to be down if it hasn't reported heartbeats.
 	MaxStoreDownDuration timeutil.Duration `toml:"max-store-down-duration" json:"max-store-down-duration"`
 
+	// ScheduleInterval is the interval to schedule.
+	ScheduleInterval timeutil.Duration `toml:"schedule-interval" json:"schedule-interval"`
 	// LeaderScheduleLimit is the max coexist leader schedules.
 	LeaderScheduleLimit uint64 `toml:"leader-schedule-limit" json:"leader-schedule-limit"`
-	// LeaderScheduleInterval is the interval to schedule leader.
-	LeaderScheduleInterval timeutil.Duration `toml:"leader-schedule-interval" json:"leader-schedule-interval"`
-
-	// StorageScheduleLimit is the max coexist storage schedules.
-	StorageScheduleLimit uint64 `toml:"storage-schedule-limit" json:"storage-schedule-limit"`
-	// StorageScheduleInterval is the interval to schedule storage.
-	StorageScheduleInterval timeutil.Duration `toml:"storage-schedule-interval" json:"storage-schedule-interval"`
-
+	// RegionScheduleLimit is the max coexist region schedules.
+	RegionScheduleLimit uint64 `toml:"region-schedule-limit" json:"region-schedule-limit"`
 	// ReplicaScheduleLimit is the max coexist replica schedules.
 	ReplicaScheduleLimit uint64 `toml:"replica-schedule-limit" json:"replica-schedule-limit"`
-	// ReplicaScheduleInterval is the interval to schedule storage.
-	ReplicaScheduleInterval timeutil.Duration `toml:"replica-schedule-interval" json:"replica-schedule-interval"`
 }
 
 const (
-	defaultMaxReplicas             = uint64(3)
-	defaultMinRegionCount          = uint64(10)
-	defaultMinLeaderCount          = uint64(10)
-	defaultMaxSnapshotCount        = uint64(3)
-	defaultMinBalanceDiffRatio     = float64(0.01)
-	defaultMaxStoreDownDuration    = time.Hour
-	defaultLeaderScheduleLimit     = 8
-	defaultLeaderScheduleInterval  = time.Second
-	defaultStorageScheduleLimit    = 4
-	defaultStorageScheduleInterval = time.Second
-	defaultReplicaScheduleLimit    = 8
-	defaultReplicaScheduleInterval = time.Second
+	defaultMaxReplicas          = uint64(3)
+	defaultMinRegionCount       = uint64(10)
+	defaultMinLeaderCount       = uint64(10)
+	defaultMaxSnapshotCount     = uint64(3)
+	defaultMinBalanceDiffRatio  = float64(0.01)
+	defaultMaxStoreDownDuration = time.Hour
+	defaultScheduleInterval     = time.Minute
```

Comment:
I don't think it's reasonable. If the limit is 1, then the schedule interval will be 60s. This will cause most of the time there is no operator running.